yarn.lock

```diff
@@ -1234,9 +1234,9 @@ balanced-match@^1.0.0:
   version "1.0.0"
```

Comment:
Did you mean to include this file in the patch? We often don't include these and then push them separately, but I don't think that needs to be a requirement.

Reply:
I did but I can see it could be easier to leave it out. 

There's a project we can setup which will update lock files for dep changes. Once we do that having to manually push lockfile changes should be a thing of the past.